### PR TITLE
Update for PolymerElements and behaviors

### PR DESCRIPTION
**Pull request by cdata**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-core-pages
+iron-pages
 ==========
 
-See the [component page](http://polymer-project.org/docs/elements/core-elements.html#core-pages) for more information.
+See the [component page](http://polymer-project.org/docs/elements/iron-elements.html#iron-pages) for more information.
```
Comment (morethanreal) [inline README.md:4]: Should we replace this with a short description?

File: demo/index.html [hunk 2]
```diff
@@ -11,13 +11,13 @@
 <html>
 <head>
 
-  <title>core-pages</title>
+  <title>iron-pages</title>
 
   <meta charset="utf-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
```
Comment (morethanreal) [inline demo/index.html]: Use this meta tag (magical invocation for HW acceleration on Android): `<meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">`

**Comment by morethanreal** [comment]
lgtm, love the behaviors!

File: README.md (hunk 1 above)
Comment (cdata) [inline README.md:4]: I don't know, do we do that anywhere else?

File: demo/index.html (hunk 2 above)
Comment (cdata) [inline demo/index.html]: Yay magic.

File: README.md (hunk 1 above)
Comment (morethanreal) [inline README.md:4]: I've been doing it for new repositories I made. I guess those links will become more useful whenever we push the doc page..

File: iron-pages.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    }
+
+  </style>
+
+  <template>
+
+    <div class="content">
+      <content></content>
+    </div>
+
```
Comment (frankiefu) [inline iron-pages.html:56]: I think we can eliminate the extra `div`, and just `<content>` in the template:

``` html
<template>

  <content></content>

</template>
```

and the style will be:

```
:host > ::content > :not(.iron-selected) {
  display: none !important;
}
```

**Pull request merged**